src/cmd.ts

```diff
@@ -78,6 +78,31 @@ function createCommandWithSharedOptions(name, description) {
 			"Category to assign torrents from data-based matching",
 			fallback(fileConfig.dataCategory, "cross-seed-data")
 		)
+		.option(
+			"--link-dir <dir>",
+			"Directory to output data-matched hardlinks to",
+			fileConfig.linkDir
+		)
+		.option(
+			"--use-hardlinks",
+			"Will use hardlinks if a link-dir is supplied instead of symlinks",
+			fallback(fileConfig.useHardlinks, false)
+		)
+		.option(
+			"--skip-recheck",
+			"Skips torrent recheck upon adding to QBittorrent",
+			fallback(fileConfig.skipRecheck, false)
+		)
+		.option(
+			"--max-data-depth <depth>",
+			"Max depth to look for searchees in dataDirs",
+			fallback(fileConfig.maxDataDepth, 2)
+		)
+		.option(
+			"--single-search",
```

Comment:
Now that I've seen daemon mode is a different path, this option is basically pointless. However keeping this as a note that the function for generating searchees from data needs to have a parameter added to accomplish the same thing for use during daemon mode.